### DISCUSSION TITLE
lion-tensorflow implementation

### DISCUSSION THREAD

**Discussion by GLambard** [lucidrains/lion-pytorch#6] [Show and tell]
Just did a Lion optimizer for Tensorflow 100% based on this repo! You can find it [here](https://github.com/GLambard/Lion-tensorflow)

Thank you so much lucidrains for your pytorch version!

**Comment by lucidrains** [comment]
@GLambard oh very nice :100: ! in turn i mostly based it from google's automl repository haha

have you tried running it on any tasks?
